spec/parse.spec.js

```diff
@@ -1328,7 +1328,35 @@ var TESTS = [
 
 			return result;
 		})()
-	}
+	},
+	{
+		input: "@layer custom-layer { div { display: block; } }",
```

Comment:
consider also adding a test for anonymous layers (layers without a name)

`@layer { div { display: block; } }`

Reply:
sure, done - in this case the model has an empty layerName